src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -10573,6 +10573,17 @@ bool Unit::IsInFeralForm() const
     return form == FORM_CAT_FORM || form == FORM_BEAR_FORM || form == FORM_DIRE_BEAR_FORM || form == FORM_GHOST_WOLF;
 }
 
+bool Unit::IsInTravelForm() const
+{
+    ShapeshiftForm form = GetShapeshiftForm();
+    return form == FORM_TRAVEL_FORM || form == FORM_AQUATIC_FORM || form == FORM_FLIGHT_FORM || form == FORM_FLIGHT_FORM_EPIC;
```

Comment:
IsTravelForm return if it's a travel form or not (called by playerscript for unaura travel if shapeshift are not a travel form)